rows.go

```diff
@@ -209,3 +210,14 @@ func (rows *snowflakeRows) waitForAsyncQueryStatus() error {
 	}
 	return nil
 }
+
+func (rows *snowflakeRows) addDownloader(newDL chunkDownloader) error {
```

Comment:
interface does not need to return `err` if all code path are returning `nil`
noticed that all callers are also not checking err return

Reply:
addressed